scripts/minikube.sh

```diff
@@ -99,9 +98,22 @@ function minikube_losetup() {
     ${minikube} ssh 'sudo sh -c "rm -f /sbin/losetup && cp ~docker/losetup /sbin"'
 }
 
+function minikube_supports_psp() {
+    local MINIKUBE_MAJOR
+    local MINIKUBE_MINOR
+    local MINIKUBE_PATCH
+    MINIKUBE_MAJOR=$(minikube_version 1)
+    MINIKUBE_MINOR=$(minikube_version 2)
+    MINIKUBE_PATCH=$(minikube_version 3)
+    if [[ "${MINIKUBE_MAJOR}" -ge 1 ]] && [[ "${MINIKUBE_MINOR}" -ge 11 ]] && [[ "${MINIKUBE_PATCH}" -ge 1 ]] || [[ "${MINIKUBE_MAJOR}" -ge 1 ]] && [[ "${MINIKUBE_MINOR}" -ge 12 ]]; then
+        return 1
+    fi
+    return 0
+}
+
 # configure minikube
 MINIKUBE_ARCH=${MINIKUBE_ARCH:-"amd64"}
-MINIKUBE_VERSION=${MINIKUBE_VERSION:-"latest"}
+MINIKUBE_VERSION=${MINIKUBE_VERSION:-"v1.12.2"}
```

Comment:
Fixed versions should only be in the `build.env` file. When trying to install minikube with `latest`, I think it should not check the currently installed minikube version, but just install it.

There might be a minikube command to check if there is an update available. That could be used in addition to the above.

Reply:
+1 @nixpanic